pom.xml

```diff
@@ -1961,27 +1965,6 @@
               </execution>
             </executions>
           </plugin>
-          <plugin>
-            <groupId>org.jacoco</groupId>
-            <artifactId>jacoco-maven-plugin</artifactId>
-            <executions>
-              <execution>
-                <goals>
-                  <goal>prepare-agent</goal>
-                </goals>
```

Comment:
this step from jacoco plugin under integration-tests profile caused mystic class loading issue in flink 1.17 cc @danny0405 

```
[ERROR] testMergeOnReadInputFormatLogFileOnlyIteratorGetUnMergedLogFileIterator  Time elapsed: 0.006 s  <<< ERROR!
java.lang.NoClassDefFoundError: Could not initialize class org.apache.calcite.rel.metadata.DefaultRelMetadataProvider
	at org.apache.hudi.table.ITTestSchemaEvolution.setUp(ITTestSchemaEvolution.java:81)
```